platform/backend/repository/testbed/src/test/kotlin/io/hamal/repository/TriggerTest.kt

```diff
@@ -272,6 +277,7 @@ internal class TriggerRepositoryTest : AbstractUnitTest() {
                     assertThat(name, equalTo(TriggerName("trigger-name")))
                     assertThat(inputs, equalTo(TriggerInputs()))
                     assertThat(topicId, equalTo(TopicId(9)))
+                    assertThat(status, equalTo(TriggerStatus.Active))
```

Comment:
```suggestion
                    assertThat(status, equalTo(Active))
```

static imports when ever possible